Makefile

```diff
@@ -14,7 +14,7 @@ all: lunatik_sym.h
 
 clean:
 	make -C ${KDIR} M=${PWD} clean
-	rm lunatik_sym.h
+	rm -f lunatik_sym.h
```

Comment:
perhaps a better idea is to put this in a var.. e.g.,
```
RM = rm -f
```

and while there do the same for mkdir, and install.. just a suggestion, I can merge as is if you disagree..

Reply:
Yes, that is much better

Reply:
I will make the changes